app/controllers/api/v1/notes_controller.rb

```diff
@@ -13,6 +17,21 @@ def show
 
       private
 
+      def new_note
+        current_user.notes.create(params.require(:note)
+                                                .permit(:title,
+                                                        :content,
+                                                        :note_type))
+      end
+
+      def render_create_message(resource)
```

Comment:
Raro que rubocop no lo marque, pero este `.permit` debería estar alineado con el `.require`
```suggestion

      def new_note
        current_user.notes.create(params.require(:note)
                                        .permit(:title, :content, :note_type))
      end

      def render_create_message(resource)
```
Muchas veces utilizamos la notación alternativa de array de simbolos con el %i, para acortar el código y dejarlo más claro